### ISSUE TITLE
npm run build with olm

### ISSUE THREAD

**Issue by dbkr** [vector-im/vector-web#1765]
Gets an error for me when olm is installed:

```
ERROR in ../matrix-js-sdk/lib/OlmDevice.js
Module not found: Error: Cannot resolve module 'olm' in /Users/dave/matrix/matrix-js-sdk/lib
 @ ../matrix-js-sdk/lib/OlmDevice.js 18:10-24
```

Also is very slow, probably because olm.js is being minified which it would be good to skip if possible.

**Comment by richvdh** [comment]
The critical information that is missing here is that the error happens when running `npm run build` in `vector-web` with Olm installed in `vector-web` and *not* `matrix-js-sdk`.

**Comment by richvdh** [comment]
So the problem is that, when `matrix-js-sdk` is symlinked, webpack only looks for `olm` in `matrix-js-sdk/node_modules`, whereas the webpack configuration script has looked in `vector-web/node_modules` and found an olm. I could probably work around this with an alias.

As for the slowness, yes it's probably uglify running on olm.js. Whilst this is of course redundant, I don't believe it's possible to turn off uglify on a single input file - it can only be done for entire webpack chunks (see https://github.com/webpack/webpack/issues/1079). We could split the vector dist up into more chunks to work around this, but that has some downsides too. In short, we might just have to put up with it.

**Comment by richvdh** [comment]
fixed by #1766, hopefs
